hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/service/TestKeyDeletingService.java

```diff
@@ -567,6 +569,45 @@ volumeName, bucketName, getSnapshotPrefix("snap3"), true)) {
 
   }
 
+  @Test
+  public void testLogContent() throws IOException, AuthenticationException,
```

Comment:
Let's don't add a test since it is just log message change.  You may manually test it.